src/main/webapp/intrigue-api/metacards/metacards.js

```diff
@@ -202,6 +203,22 @@ const lists = (id, fetch, toGraphqlName) => async () => {
     : null
 }
 
+const makeActionIdUnique = (id, action) => {
+  return {
+    ...action,
+    id: action.id + '-' + id,
+  }
+}
+
+const thumbnailUrl = result => {
+  const { url } = result.actions.find(
```

Comment:
Checking for `?transform=thumbnail` tells me if the default DDF thumbnail action is being used or overridden.  If it is used then we default to using the base64 encoded thumbnail to avoid more round trips for the metacard.  If it is a different action provider generating the URL, then we should use the URL provided by the downstream distribution.

I did not think of the `undefined` but that explains all of the broken image icons.  I will add a check for that.

Reply:
Ah, OK. 
Thank you for the explanation.